README.md

```diff
@@ -26,339 +26,213 @@
-[C/C++](https://github.com/triton-inference-server/backend/blob/main/README.md#triton-backend-api)
-or
-[Python](https://github.com/triton-inference-server/python_backend).
-Triton also allows developers to write backends that can send multiple
-responses for a request or not send any responses for a request. Backends
-and models that operate in this way are referred to as [decoupled
-backends and models](docs/decoupled_models.md).
-
-A [Triton repository agent](docs/repository_agents.md) extends Triton
-with new functionality that operates when a model is loaded or
-unloaded. You can introduce your own code to perform authentication,
-decryption, conversion, or similar operations when a model is loaded.
-
-## Additional Resources
-
-* Additional documentation, presentations, and examples are located in the
-  [NVIDIA Developer Zone](https://developer.nvidia.com/nvidia-triton-inference-server).
-
-* Specific end-to-end examples for popular models, such as ResNet, BERT, and
-  DLRM are located in the [NVIDIA Deep Learning Examples page on
-  GitHub](https://github.com/NVIDIA/DeepLearningExamples).
+configuration](docs/model_configuration.md) for the model.  
+
+- Optimize your models setting [scheduling and batching](docs/architecture.md#models-and-schedulers)
+  parameters and [model instances](docs/model_configuration.md#instance-groups).
+- Use the [Model Analyzer tool](https://github.com/triton-inference-server/model_analyzer)
+  to help optimize your model configuration with profiling
+- [Add custom operations to Triton if needed by your model](docs/custom_operations.md)
+- Enable model pipelining with [Model Ensemble](docs/architecture.md#ensemble-models)
+  and [Business Logic Scripting (BLS)](https://github.com/triton-inference-server/python_backend#business-logic-scripting)
+- Learn how to [explicitly manage what models are available by loading and 
+  unloading models](docs/model_management.md)
+
+#### Configure and Use Triton Inference Server
+
+- Read the [Quick Start Guide](docs/quickstart.md) on to run Triton Inference Server
+  on both GPU and CPU
+- Triton supports multiple execution models, called 
+  [backends](https://github.com/triton-inference-server/backend), including 
+  [TensorRT](https://github.com/triton-inference-server/tensorrt_backend), 
+  [TensorFlow](https://github.com/triton-inference-server/tensorflow_backend), 
+  [PyTorch](https://github.com/triton-inference-server/pytorch_backend), 
+  [ONNX](https://github.com/triton-inference-server/onnxruntime_backend), 
+  [OpenVINO](https://github.com/triton-inference-server/openvino_backend), 
+  [Python](https://github.com/triton-inference-server/python_backend), and more
+- Learn how to [optimizing performance](docs/optimization.md) using the 
+  [Performance Analyzer](docs/perf_analyzer.md) and 
+  [Model Analyzer](https://github.com/triton-inference-server/model_analyzer)
+- Learn how to [manage loading and unloading models](model_management.md)
```

Comment:
Learn how to [manage loading and unloading models](model_management.md) _in Triton_